File: app/Http/Controllers/SpecialistPatterns/LR/LR1SP/graph/Coords.php
<?php

namespace App\Http\Controllers\SpecialistPatterns\LR\LR1SP\graph;

// Класс Coords представляет координаты (x, y).
class Coords {
    // Приватные свойства для хранения координат x и y.
    private int $x;
    private int $y;

    // Конструктор класса. Инициализирует объект координатами (x, y).
    public function __construct(int $x, int $y) {
        $this->x = $x;
        $this->y = $y;
    }

    // Метод для получения значения координаты x.
    public function getX(): int {
        return $this->x;
    }

    // Метод для установки значения координаты x.
    public function setX(int $x): void {
        $this->x = $x;
    }

    // Метод для получения значения координаты y.
    public function getY(): int {
        return $this->y;
    }

    // Метод для установки значения координаты y.
    public function setY(int $y): void {
        $this->y = $y;
    }
}

File: app/Http/Controllers/SpecialistPatternsJava/Lab1/Lab1Java/src/graph/Coords.java
package graph;

// Класс Coords представляет собой объект для хранения координат (x, y).
public class Coords {
    private int x;
    private int y;

    // Конструктор класса, инициализирующий координаты (x, y).
    public Coords(int x, int y) {
        this.x = x;
        this.y = y;
    }

    // Метод для получения значения координаты X.
    public int getX() {
        return x;
    }

    // Метод для установки значения координаты X.
    public void setX(int x) {
        this.x = x;
    }

    // Метод для получения значения координаты Y.
    public int getY() {
        return y;
    }

    // Метод для установки значения координаты Y.
    public void setY(int y) {
        this.y = y;
    }
}
